.taskcluster.yml

```diff
@@ -71,7 +60,7 @@ tasks:
       in:
           $if: >
               tasks_for in ["action", "cron"]
-              || (tasks_for == "github-push" && head_branch == "refs/heads/main")
+              || (tasks_for == "github-push" && (head_branch == "refs/heads/main" || head_branch[:15] == "refs/heads/dev-"))
```

Comment:
In the routes section where I'm removing the level 3 check for the treeherder route? If so, I don't think so...otherwise we won't get treeherder symbols for these tasks in the staging repository. (I have a feeling I'm misunderstanding this comment though...)

Reply:
No, just below on line 68 (annoyingly Github doesn't let me comment on unmodified lines). Or is the intent for these branches to be level 3? (If so then carry on)

Reply:
They'll still be level 1 on staging-firefox-translations-training due to the repoUrl check, but I was assuming we don't want to be running these as L3 on the main repo either.